projects/smoketest_max6643/src/main.c

```diff
@@ -0,0 +1,58 @@
+#include <stdbool.h>
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "max6643_fan_controller.h"
+#include "wait.h"
+
```

Comment:
Generally we put a comment at the start of smoke tests explaining how to use it and what each of the configurable items (i.e. the defines) mean. This helps the hardware person using the smoke test to validate the board to understand how to use the smoke test.